python-mini-projects/projects/Create_a_simple_stopwatch/stopwatch.py

```diff
@@ -0,0 +1,81 @@
+import tkinter as Tkinter
+from datetime import datetime
+counter = 0
+running = False
+
+
+def counter_label(label):
+    def count():
+        if running:
+            global counter
+			# To manage the intial delay. 
+            if counter == 0:
+                display = 'Ready!'
+            else:
+                tt = datetime.utcfromtimestamp(counter)
+                string = tt.strftime('%H:%M:%S')
+                display = string
+	
+            label['text'] = display
+	
+			# label.after(arg1, arg2) delays by 
+			# first argument given in milliseconds 
+			# and then calls the function given as second argument. 
+			# Generally like here we need to call the 
+			# function in which it is present repeatedly. 
+			# Delays by 1000ms=1 seconds and call count again. 
+            label.after(1000, count)
+            counter += 1
+	
+	# Triggering the start of the counter. 
+    count()
+	
+
+# start function of the stopwatch 
+def Start(label):
+    global running
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Global variables are dangerous because they can be simultaneously accessed from multiple sections of a program. This frequently results in bugs. Most bugs involving global variables arise from one function reading and acting on the value of a global variable before another function has the chance to set it to an appropriate value. We recommend using a configuration module to mutate global state.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules)

Similar issue at line numbers 46 and 55.